8th tutorial video review (#166)
w3c/wot-cg: Tutorials/whatiswot/8-JSON_Schema_in_Practice_3/8-JSON-Schema-3.txt

In the previous videos of JSON Schema, we have explained the features and basic implementations of JSON Schema. In this video, we will show the actual validation process with libraries and code examples.

> Throughout this video, we will mainly focus on validation with the Python library jsonschema and the JavaScript library AJV, but there are many other libraries in different programming languages and tools to validate JSON using a JSON Schema. 

Reviewer: ```suggestion
Throughout this video, our main focus will be on validation with the `jsonschema` Python library and the `AJV` JavaScript library, but many other libraries to validate JSON using a JSON Schema exist in different programming languages and tools. 
```

> Here are some of the popular ones in different programming languages. There are many more validators.  

Reviewer: ```suggestion
There are many validators. Here are some of the more popular ones.  
```

For further information, you can check the JSON Schema homepage via the link in the video description.

This  is the basic validation flow that all libraries follow.

> The library takes a schema for your JSON data and validates your data according to the schema. Outputs the validation result as a boolean  

Reviewer: ```suggestion
The library takes a schema for your JSON data and validates your data according to the schema. The validation result is output as a boolean.
```

So let's start with AJV! It is a JavaScript library.

This is the same temperature sensor example we used before. Let’s examine this in JavaScript code.

> First, We are importing the AJV.  

Reviewer: ```suggestion
First, we import the `AJV` library.  
```
